ITC205_BaseLine_Code/src/PayFineControl.java

```diff
@@ -35,31 +34,29 @@ public void cardSwiped(int memberId) {
 			return;
 		}
 		ui.display(member.toString());
-		ui.setState(PayFineUI.UI_STATE.PAYING);
-		state = CONTROL_STATE.PAYING;
+		ui.setState(PayFineUI.UIState.PAYING);
+		state = ControlState.PAYING;
 	}
 	
 	
 	public void cancel() {
-		ui.setState(PayFineUI.UI_STATE.CANCELLED);
-		state = CONTROL_STATE.CANCELLED;
+		ui.setState(PayFineUI.UIState.CANCELLED);
+		state = ControlState.CANCELLED;
 	}
 
-
+	
 	public double payFine(double amount) {
-		if (!state.equals(CONTROL_STATE.PAYING)) {
+		if (!state.equals(ControlState.PAYING)) {
 			throw new RuntimeException("PayFineControl: cannot call payFine except in PAYING state");

```

Comment:
D5 - Put a line break in after PayFineControl:

**this is because that line exceeds the 100 character limit

Reply:
Fixed